chart/identity-manager-sts/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "common.names.fullname" . }}
+spec:
+  ports:
+    - name: web
+      port: 80
+      protocol: TCP
+      targetPort: 8080
```

Comment:
Consider making these a template value or helper value for both service spec and pod spec to ensure there isn't a typo later on